### PR TITLE
[DE] Update `Help centre/Store/Store archive`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
The title should be translated.

Related to #12147

- [x] Check if #12388 makes any changes in `Store archive`

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Help_centre/Store/Store_archive/de.md
Comment (Y2nn1c) [inline wiki/Help_centre/Store/Store_archive/de.md:18]:
```suggestion
*Anmerkung: Diese Produkte werden nicht mehr zum Verkauf im osu!store angeboten.*
```
idk if we should keep osu!store as a term or change it to something like osu!store, osu!laden...

Comment (The-Last-Cookie) [reply wiki/Help_centre/Store/Store_archive/de.md:18]: According to the [Brand identity guidelines](https://osu.ppy.sh/wiki/de/Brand_identity_guidelines), `osu!store` is a protected name and must not include spaces.

I would like to stick to the original name where possible due to this.

Comment (Y2nn1c) [reply wiki/Help_centre/Store/Store_archive/de.md:18]: > According to the [Brand identity guidelines](https://osu.ppy.sh/wiki/de/Brand_identity_guidelines), `osu!store` is a protected name and must not include spaces.
> 
> I would like to stick to the original name where possible due to this.

Sounds good to me. The context should be enough for anybody to understand what it is although we could turn osu!store into a link to https://osu.ppy.sh/store/listing

Comment (The-Last-Cookie) [reply wiki/Help_centre/Store/Store_archive/de.md:18]:
```suggestion
*Anmerkung: Diese Produkte werden nicht mehr zum Verkauf im osu!store angeboten.*
```

**Comment by The-Last-Cookie** [comment]
Waiting for the review by Yannic2112 to come back (his account is [likely temporarily] banned)

**Comment by The-Last-Cookie** [comment]
> The context should be enough for anybody to understand what it is although we could turn osu!store into a link to https://osu.ppy.sh/store/listing

Debatable in my opinion. Users can already access the store via the menu header at the top, and this change is out of scope here anyway because every language would need to adjusted.

**Review by Y2nn1c — APPROVED** [review]
Looks good. No mistakes in the writing and everything has been translated properly.
I think it's ready to be shipped in it's current state.

**Review by Hiviexd — APPROVED** [review]

**Pull request merged**
